docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,107 @@
+# Move an instance from one Cleura Cloud region to another
+
+This guide will show you how to move or copy your instance to a different region in Cleura cloud.
+
+## Prerequisites
+
+- Installed OpenStack [CLI client](https://docs.cleura.cloud/howto/getting-started/enable-openstack-cli/)
+- Access to source and target location (RC-file)
+- Enough space on the local machine that you download your server to.
+- Target location configured with Network, Security Groups etc
+
+>To speed up this work, we recommend to use a temporary linux instance located at the same region as the source. This make the transfer faster not needing to downloading to and uploading from your local machine.
+
+## Recommendations
+
+First step will be to create a snapshot of the instance, during this step it is recommended to shutdown the instance to ensure data consistency. Once the snapshot is done the instance can be rebooted again.
```

Comment:
* _The_ first step.
* The comma is wrong; it should either be a semicolon, or the sentence should be split in two.
* Active voice is better than passive: "Once you have completed the snapshot, you can restart the instance."